Source/WebKit/Platform/IPC/StreamClientConnection.h

```diff
@@ -241,7 +241,7 @@ std::optional<StreamClientConnection::SendSyncResult<T>> StreamClientConnection:
             if (!replySpan)
                 return std::unique_ptr<Decoder> { };
             auto decoder = std::unique_ptr<Decoder> { new Decoder(replySpan->data, replySpan->size, m_currentDestinationID) };
-            if (decoder->messageName() != MessageName::ProcessOutOfStreamMessage) {
+            if (decoder->messageName() != MessageName::ProcessOutOfStreamMessage && decoder->messageName() != MessageName::SetStreamDestinationID) {
```

Comment:
On the surface level this looks like some sort of corruption.

Reply:
I'm investigating why this is occurring. It occurs during the call to RemoteGPUProxy::RequestAdapter:
```
auto sendResult = sendSync(Messages::RemoteGPU::RequestAdapter(*convertedOptions, identifier));
```